configuration/openmrs/apps/clinical/extension.json

```diff
@@ -134,18 +134,44 @@
     "requiredPrivilege": "app:clinical"
   },
   "bahmniClinicalPatientsSearchMalnutritionPatient": {
-    "id": "bahmni.clinical.patients.search.malnutritionPatients",
+    "id": "bahmni.clinical.patients.search.MalnutritionPatients",
     "extensionPointId": "org.bahmni.patient.search",
     "type": "config",
     "extensionParams": {
-      "searchHandler":"emrapi.sqlSearch.malnutritionPatients",
+      "searchHandler": "emrapi.sqlSearch.MalnutritionPatients",
```

Comment:
Correct syntaxt would be camelCase here: `malnutritionPatients`
I don't think this has any implications though.

Reply:
Let's wait for the change of the Global Property name on openmrs-config-c2c